src/udp/connection/timestamp_64.rs

```diff
@@ -1,9 +1,3 @@
 /// Connection id contains a timestamp in 4 bytes due to its 64 bits limit.
 pub type Timestamp64 = u64;
```

Comment:
hey @WarmBeer I think I'm going to change this type too to `pub struct Timestamp32(pub u32);`

It requires an extra conversion from the clock timestamp but I prefer the symmetry :-)